src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx

```diff
@@ -287,6 +292,19 @@ const ObjectDetailPage: NextPage = () => {
 					</div>
 				</article>
 			</div>
+			<AddToCollectionBlade
+				isOpen={activeBlade === MediaActions.Bookmark}
+				selected={{
+					id: mediaInfo?.id ?? '',
+					title: mediaInfo?.name,
+				}}
+				onClose={() => {
+					setActiveBlade(undefined);
```

Comment:
Deze in een aparte `const onCloseBlade = () => setActiveBlade(null);` en best met `null` ipv `undefined` werken als je met een lege value wil werken.

Reply:
Werkt op deze manier omdat er in de toekomst nieuwe blades kunnen bijkomen en er dan geen nieuwe state bijgehouden moet worden.